spec/models/post.spec.js

```diff
@@ -24,15 +47,15 @@ describe("Post model", () => {
   });
 
   it("can save a post", (done) => {
-    var post = new Post({ message: "some message" });
+    var post = new Post({ message: "can save a post", user: mockUserId });
```

Comment:
You've updated most of the references from var to const - would be worth catching this one too

Reply:
Resolved in newest commit